### ISSUE TITLE
User Guide Review

### ISSUE THREAD

**Issue by Rachelwrq** [OpenLiberty/draft-guide-arquillian-managed#27]
#### Running the tests
- [ ] Don't know if this is necessary but most of the guide I reviewed have test result in this section; might want to ad it for consistency. 

### User Testing Checklist
- [ ] Record the time it takes to go through the entire guide
- [ ] Test each step in the guide is clear to understand
- [ ] Ensure no logic errors 
- [ ] Resolve anything that does not work
- [ ] Resolve anything usability issues (confusing or hard to follow)
- [ ] Achieve basic understanding/working knowledge of the topic in the guide

**Comment by evelinec** [comment]
This guide is different than the other guides in terms of the test results. The expected results were described in text, but no meaningful results can be shown.

**Comment by evelinec** [comment]
Thank you for your review and testing! @Rachelwrq
